### DISCUSSION TITLE
Network Error on Login Screen - Fullstack Template gitclone and copier setup approach

### DISCUSSION THREAD

**Discussion by MicahPearce** [fastapi/full-stack-fastapi-template#1368] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
py -m  copier copy https://github.com/fastapi/full-stack-fastapi-template Limber-Dash --trust
fnm install
fnm use
npm install
npm run dev 

Network error
```

### Description

I have done a clean setup following both the github clone and the copier guide and keep hitting the following breaking error.

After completing initial setup of the template I run npm run dev and everything appears to be working fine and I can navigate to the login page

![image](https://github.com/user-attachments/assets/0ae59b48-b018-4d85-b77d-322fb0cbf769)

However when attempting to login to the dashboard I get 'Network Error' when logging in with the firstsuperuser creds or any dummy use creds - other discussions related to this issue seem to focus on complications when modifying something to use on docker however I am fully running locally and have done no customization to any files 

![image](https://github.com/user-attachments/assets/c046653e-19a1-4047-ad44-e35182ec7378)

![image](https://github.com/user-attachments/assets/7decfd7c-4b57-42e7-ba47-9285983476bd)

### Operating System

Windows

### Operating System Details

Windows 11 23H2

### Python Version

Python 3.12.6

**Comment by ikiteiru** [comment]
Same. Up

**Comment by etanase** [comment]
Send the output of the backend container, see if it is up and receiving any requests

**Comment by RomaCZ** [comment]
Yes, i'we been there too 😉
Just because you r running front end with npm run dev and prolly backend with fastapi Its not enought.
You still need to run ducker postgres database. Otherwise you get tis error. So fireup docker, usebthat docker copose to run all of it. Than you can turn off front end/backend and start hacking locally

**Comment by lxbworld** [comment]
I meet this trouble as well. You can try this: 1. turn off vpn; 2. restart the whole project, then it will work. Maybe something need taking time to start.

**Comment by YuriiMotov** [comment, marked as answer]
Before running frontend, you should run `docker compose watch`, then stop frontend container by `docker compose stop frontend`.

Docs: https://github.com/fastapi/full-stack-fastapi-template/blob/master/development.md
